.github/workflows/e2e.yml

```diff
@@ -0,0 +1,122 @@
+name: e2e
+
+on: [push]
+env:
+  REGISTRY: 'ghcr.io/dell/csi-baremetal'
+  go_version: '1.16.8'
+  golangci_version: '1.37.0'
+  helm_version: '3.7.1'
+  kubectl_version: 'v1.22.2'
+jobs:
+  e2e:
+    runs-on: ubuntu-20.04
+    if: "contains(github.event.head_commit.message, '[e2e]')"
+    steps:
+    - name: Check out code
+      uses: actions/checkout@v2
+      
+    - name: Add csi-baremetal-operator
+      uses: actions/checkout@v2
+      with:
+        repository: dell/csi-baremetal-operator 
```

Comment:
Is it possible to test with specific branch or commit ID? If you have changes in both repos CI will never pass...

Reply:
We could make it with triggering CI via PR comment, it requires similar changes